src/components/GiveProject/GrantCard.tsx

```diff
@@ -242,7 +207,7 @@ export default function GrantCard({ grant, mode }: GrantDetailsProps) {
                     <Icon name="donors" />
                   </Grid>
                   <Grid item className="metric">
-                    {donorCountIsLoading ? <Skeleton className="skeleton-inline" /> : donorCount}
+                    {!donorCount ? <Skeleton className="skeleton-inline" /> : donorCount}
```

Comment:
`useDonationInfo().isLoading` is what you should be accessing here.

If there are no donors to a project, `!donorCount` will return `true` and you'll show the skeleton component.